src/rest/subhandlers/rest_edges.erl

```diff
@@ -10,11 +10,10 @@
 %% API
 -export([
     init/2,
-    content_types_provided/2,
-    content_types_accepted/2,
-    resource_exists/2,
     allowed_methods/2,
-    is_conflict/2
+    content_types_accepted/2,
```

Comment:
```suggestion
    accepted_content_types/2,
```

Reply:
Nope. To tak nie działa - to jest obowiązkowy cowboy-owy callback. Ogólnie w handlerach jest jest tak, że pierwszy export zawiera obowiązkowe callbacki cowboy-a które muszą się nazywać tak jak się nazywają